pwndbg/dt.py

```diff
@@ -100,22 +100,29 @@ def dt(name='', addr=None, obj = None):
     if addr is not None:
         obj = pwndbg.memory.poi(t, addr)
 
-    # Header, optionally include the name
-    header = name
-    if obj: header = "%s @ %s" % (header, hex(int(obj.address)))
-    rv.append(header)
-
     if t.strip_typedefs().code == gdb.TYPE_CODE_ARRAY:
         return "Arrays not supported yet"
     if t.strip_typedefs().code not in (gdb.TYPE_CODE_STRUCT, gdb.TYPE_CODE_UNION):
         t = {name: obj or gdb.Value(0).cast(t)}
 
+    # Header, optionally include the name
+    header = name
+    if obj: header = "%s @ %s" % (header, hex(int(obj.address)))
```

Comment:
Lets make it more simple: 
```python
'%s @ 0x%x' % (heaer, int(obj.address))
```

 I wonder if `int()` is needed at all.

Reply:
#yolo